data/gui/window/network_transmission.cfg

```diff
@@ -95,7 +95,6 @@
 									definition = "gold_small"
 									# NOTE: needed to avoid explicit calls to invalidate_layout()
```

Comment:
This leaves a dangling comment that makes no sense without the line that it referred to.